### PR TITLE
Added a showcase area for new updates and notifications

### PR DESCRIPTION
**Pull request by ankushdharkar**
This area will be useful to show some of the new things happening across our sites

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,9 +19,17 @@
         </a>
     </nav> 
 
+    <section class="interesting-notifications">
+      <p class="notif">
+        <span>New</span>
+        <strong>Crypto</strong>
+        -- New updates to the user dashboard
+        <a href="https://crypto.realdevsquad.com"> Check it out! ➡️ </a>
```
Comment (swarajpure) [inline index.html:27]: What if we take the arrow emoji out of anchor tag?
The underline for anchor tag is going to the right of arrow emoji and it is looking a little weird. I initially thought it's a dot.
image

**Review by swarajpure — COMMENTED** [review]
Thanks for this feature! 😄 
Have some minor doubts. Please check.

File: css/main.css [hunk 2]
```diff
@@ -41,16 +41,45 @@ nav {
...
+}
+
+.interesting-notifications > .notif span {
+  background: linear-gradient(90deg,#e30162 25%,#ed579c 90%);
+  color: white;
+  padding: 2px 8px;
+  font-weight: bold;
+  font-size: 0.8em;
+  margin-right: 5px;
+  border-radius: 4px;
```
[the remark this answers is not in the document]
Comment (ankushdharkar) [reply css/main.css:65]: Makes sense. Thanks @swarajpure

File: index.html (hunk 1 above)
Comment (ankushdharkar) [reply index.html:27]: This is how my emoji looks: 
image

Comment (ankushdharkar) [reply index.html:27]: I want the emoji to be clickable

Comment (swarajpure) [reply index.html:27]: Even for emoji in your case, there's a little dot at the bottom right of the emoji. It's looking a bit weird to me. Plus, most of the users will come from Windows, it is looking a bit weird on Windows. Can we add a different id to it, still inside anchor tag but removing it's underline from CSS?

Just my opinion. Maybe we could compare the two - emoji not underlined and emoji underlined and see which one looks better?

Comment (ankushdharkar) [reply index.html:27]: I changed the underline to none, and using a color transition now to denote that the link is clickable

**Comment by ankushdharkar** [comment]
image

File: index.html (hunk 1 above)
Comment (swarajpure) [reply index.html:27]: Cool. Thank you!

**Review by swarajpure — APPROVED** [review]
All looks good to me now. I had suggested some changes which were fixed.

**Pull request merged**